packages/main/src/Select.ts

```diff
@@ -1174,6 +1174,20 @@ class Select extends UI5Element implements IFormInputElement {
 		return ids.length ? ids.join(" ") : undefined;
 	}
 
+	get accessibilityInfo() {
+		const description = this.ariaLabelText
```

Comment:
Maybe you find this version more readable and more in line with functional programming principles
`const description = [this.ariaLabelText, this.text].filter(Boolean).join(" ");`